quadrat/child-theme.json

```diff
@@ -389,21 +389,21 @@
 			"h4": {
 				"typography": {
 					"fontFamily": "var(--wp--preset--font-family--dm-sans)",
-					"fontSize": "20px",
+					"fontSize": "22px",
 					"lineHeight": 1.4
 				}
 			},
 			"h5": {
 				"typography": {
 					"fontFamily": "var(--wp--preset--font-family--dm-sans)",
-					"fontSize": "18px",
+					"fontSize": "20px",
 					"lineHeight": 1.4
 				}
 			},
 			"h6": {
 				"typography": {
 					"fontFamily": "var(--wp--preset--font-family--dm-sans)",
-					"fontSize": "16px",
+					"fontSize": "18px",
```

Comment:
This should use the variable `var(--wp--preset--font-size--small)`

Reply:
Fixed in https://github.com/Automattic/themes/pull/4683/commits/ced4e066263f375c82add6f20754151878155c4b